src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-100000.mdx

```diff
@@ -0,0 +1,76 @@
+---
+subject: 'Python agent'
+releaseDate: '2024-08-08'
+version: 10.0.0
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Remove Python2.7 support', 'Add Kafka server metric', 'Add capturing of memcache host and port', 'Add support for kafka-python-ng', 'Add support for uvicorn_workers', 'Automatically detect function signature in LangChasimilarity search', 'Add environment variable for GC runtime metrics']
+bugs: ['Fix bug in graphql entity name detection', 'Fix issue in LangChain where a thread is started and the transaction context is lost', 'Fix parsing of request and response when running OpenAI via LangChain', 'Instrument new path to OpenAI chat class', 'Fix pyscopg v3 API incompatibilities']
+security: []
+---
+
+## Notes
+
+This release of the Python agent drops support for Python2.7, adds a Kafka server metric, capturing of memcache host and port, support for [kafka-python-ng](https://pypi.org/project/kafka-python-ng/), support for [uvicorn_worker](https://pypi.org/project/uvicorn-worker/), an environment variable for GC runtime metrics, and automatically detects the function signature in [LangChain](https://pypi.org/project/langchain/) `similarity_search`. It also fixes a bug in [graphql](https://pypi.org/project/graphql/) entity name detection, an issue in [LangChain](https://pypi.org/project/langchain/) where a thread is started and the transaction context is lost, parsing of the request and response when running [OpenAI](https://pypi.org/project/openai/) via [LangChain](https://pypi.org/project/langchain/), [pyscopg](https://pypi.org/project/psycopg/) v3 API incompatibilities, and instruments a new path to the [OpenAI](https://pypi.org/project/openai/) chat class.
```

Comment:
It looks like Python2.7 is officially written out as Python 2.7 with a space between the language and version so we may want to update that throughout these release notes: (ex: https://www.python.org/download/releases/2.7/)